src/SprykerEco/Zed/Heidelpay/Persistence/Propel/Schema/spy_heidelpay.schema.xml

```diff
@@ -11,20 +11,20 @@
         <column name="id_payment_heidelpay" required="true" type="INTEGER" autoIncrement="true" primaryKey="true"/>
         <column name="fk_sales_order" required="true" type="INTEGER"/>
 
-        <column name="id_payment_reference" required="false" type="VARCHAR"/>
-        <column name="id_payment_registration" required="false" type="VARCHAR"/>
-        <column name="id_basket" required="false" type="VARCHAR"/>
+        <column name="id_payment_reference" required="false" type="VARCHAR" size="255"/>
```

Comment:
These changes look like a BC break because we are adding a field limitation that was not here before.

Reply:
This is a part of implementing MariaDB compatibility. Same fix should be applied for ALL varchar fields. There is an agreement that we release such fixes as minor as well as we did in core modules.

Reply:
We removed from this PR all size changes because it's doesn't align with the current bug. It was here just because of CI validation issues. But It was fixed into another PR which already merged.